src/app/workflow/workflow.component.html

```diff
@@ -45,6 +45,7 @@ <h2 class="text-center">The workflow linked to could not be found!</h2>
   <div class="row m-1" *ngIf="!showRedirect" fxLayout.lt-sm="column" fxLayout.gt-sm="row">
     <app-json-ld [json]="schema"></app-json-ld>
     <div fxFlex="80">
+      <div *ngIf="workflow.archived"><span class="bubble mt-3">Archived</span></div>
```

Comment:
I feel like we should go with a yellow banner similar to GitHub so it's easier to notice, especially for users who may be browsing public workflows

![image](https://github.com/dockstore/dockstore-ui2/assets/25287123/e31412f9-9ee7-43ff-976c-d82be0525859)